app/views/internal/response_templates/edit.html.erb

```diff
@@ -0,0 +1,3 @@
+<h1>Editing <%= @response_template.title %></h1>
```

Comment:
This should be an h2 to be consistent with other views and maintain the hierarchy.